content/en/platform/corda/5.0/deploying-operating/config/tls-certification.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "TLS Certification Installation on REST Worker"
+version: 'Corda 5.0'
+date: '2023-05-29'
+menu:
+  corda5:
+    identifier: corda5-tls-certification-installation
+    parent: corda5-cluster-config
+    weight: 3075
+section_menu: corda5
+---
+
+# TLS Certification Installation on REST Worker
+
+The Corda 5 GA release the TLS certificate on REST Worker is presented to a client any time HTTPS connection is made.
+Before, a self-signed certificate shipped with the REST Worker. If no specific parameters are provided, the self-signed certificate will be used as a default option such that the connection to the REST Worker is always HTTPS. However, a warning will be emitted into the REST log explaining how to provide parameters for custom TLS certificates.
+There are a few ways that a TLS certificate can be made available to the REST Worker. For either method there are three pieces of information required to install a valid TLS certificate.
+
+1. The TLS certificate itself is signed by a Certification Authority (CA) or an intermediary;
+2. A private key corresponding to the public key is included into the TLS certificate;
+3. The Certification Chain leads up to CA
+
+## Certificate Information Provided in PEM Format
+
+REST Worker now accepts optional command line arguments: 
```

Comment:
I asked Victor about this and he said: These should be fairly familiar terms to any enterprise software operator/DevOps engineer.
As the documentation says:
"TLS certificate"
"private key"
I am not sure how much more clear we can make this. The only idea comes to mind is to include links to Wikipedia articles..."  I can add the links if you think it will make it clearer?